Resources/Locale/en-US/_Funkystation/chemexpansion.ftl

```diff
@@ -0,0 +1,62 @@
+reagent-name-libital = Libital
+reagent-desc-libital = A Common, but weak medicine used to heal brute trauma. Commonly added to other medicine due to it's low side effects.
+
+reagent-name-probital = Probital
+reagent-desc-probital = A common analgesic which is effective at treating general brute trauma. It's useful for all types of beatings. Known to make movement slower. 
+
+reagent-name-salicylicacid = Salicylic Acid
+reagent-desc-salicylicacid = A heavy-duty analgesic that stabilizes the severely beaten, but is barely effective on people with less wounds. Overdosing will reverse the effect however.
+
+reagent-name-aiuri = Aiuri
+reagent-desc-aiuri = A Common, but weak medicine used to heal brute injuries. Commonly added to other medicine due to it's low side effects.
+
+reagent-name-lenturi = Lenturi
+reagent-desc-lenturi = A common chemical that is effective at treating burn damage, but uses water inside your body, thus dehydrating you.
+
+reagent-name-oxandrolone = Oxandrolone
+reagent-desc-oxandrolone = A heavy-duty chemical that stabilizes the severely burned, but is barely effective on people with less wounds. Overdosing will reverse the effect however.
+
+reagent-name-salbutamol = Salbutamol
+reagent-desc-salbutamol = A heavy-duty chemical that stabilizes the suffocated and reverses the effect of bloodloss, but is ineffective on people with less wounds. Overdosing will reverse the effect however.
+
+reagent-name-convermol = Convermol
+reagent-desc-convermol = A Fast-Acting asphyxiation medicine that has the side effect of putting heavy strain on the body, dose carefully.
+
+reagent-name-multiver = Multiver
+reagent-desc-multiver = A Common antitoxin used to treat poison. Cannot purge toxins compared to Pentetic Acid.
+
+reagent-name-penteticacid = Pentetic Acid
+reagent-desc-penteticacid = A complicated anti-toxin solution that quickly flushes out toxins and heals the damage left behind. Overdose causes extreme pain.
+
+reagent-name-granibitulari = Granibitulari
+reagent-desc-granibitulari = A Commonly sold medicine for all types of injuries, but extremely slow acting.
+
+reagent-name-heparin = Heparin
+reagent-desc-heparin = A Fast-Acting toxin that quickly opens wounds on a body, making them bleed rapidly. 
+
+reagent-name-rhigoxane = Rhigoxane
+reagent-desc-rhigoxane = That looks so cold and refreshing... don't take a sip, it's pretty warm... and you shall be cold as you thought it was if ingested.
+
+reagent-name-spewium = Spewium
+reagent-desc-spewium = Even smelling it warrants a vomit comet.
+
+reagent-name-pestiline = Pestiline
+reagent-desc-pestiline = This exotic and strange toxin is rumored to be the essence of sickness itself.
+
+reagent-name-exoticreagent = Exotic Reagent
+reagent-desc-exoticreagent = You've never even heard of it before, unless you are a chemist. Then, it's secrets are in your hands, and it's time to use the true power of chemistry.
```

Comment:
how about "A precursor to many different chemicals, allowing you to unlock the next level of medical science."